src/app/compare/CompareItem.tsx

```diff
@@ -36,6 +40,8 @@ export default function CompareItem({
   adjustedItemStats?: DimAdjustedItemStat;
   isInitialItem: boolean;
 }) {
+  const itemInfos = useSelector(itemInfosSelector);
+  const itemNotes = getNotes(item, itemInfos);
```

Comment:
It'd be slightly nicer to do:

```
const itemNotes = useSelector((state:RootState) => getNotes(item, itemInfosSelector(state));
```

That'll only re-render if the notes changes, rather than anytime itemInfos changes. Not really a big deal, but it's selecting what you're really after.

Reply:
tyvm